torch/ao/quantization/pt2e/prepare.py

```diff
@@ -92,6 +92,16 @@ def _maybe_insert_input_observer_for_arg_or_kwarg(
             new_obs_node = _insert_obs_or_fq(
                 arg, arg_as_input_act_obs_or_fq, model, named_modules, model.graph)  # type: ignore[arg-type]
             new_arg = new_obs_node
+            # When quantizing two layers with different configs we can have
```

Comment:
can you add a test for this? surprised that this is not caught earlier..

Reply:
so didnt add separate test because the PR after this one fails for exactly this reason. Also we havent tested quantizing a different layers with different configs (which is what the next diff does). Hence did not catch it.

Reply:
OK sounds good